### DISCUSSION TITLE
Redirecting from formulate outside homebrew/core

### DISCUSSION THREAD

**Discussion by Lesmiscore** [Homebrew/discussions#3177] [Tap maintenance and Homebrew development] [answered]
I was asked for how to redirect one formulae outside `homebrew/core` to the same one in `homebrew/core`.

The real situation is, that they have `yt-dlp/taps/yt-dlp` at their own tap, and they want to migrate to `homebrew/core/yt-dlp`. As they provide the same thing, read the same config, just redirecting the formulae satisfies their requirement.

I checked [this docs at brew.sh](https://docs.brew.sh/Rename-A-Formula), but does it apply in this case? So I only need to add a line at `formula_renames.json`?

**Comment by carlocab** [comment, marked as answer]
If you want to migrate users who have `yt-dlp/taps/yt-dlp` installed to `homebrew/core/yt-dlp` instead, it suffices to delete `yt-dlp/taps/yt-dlp` and then add it to `tap_migrations.json` in `yt-dlp/taps`.

You can look at `tap_migrations.json` in `homebrew/core` for an example.

**Reply by Lesmiscore** [reply]
Thank you so much!
